training/views.py

```diff
@@ -109,7 +112,11 @@ def get_filter_data(request):
         states_list = State.objects.annotate(value=F('state_name')).values('id','value').order_by('value')
         state_dict = {'name':'State', 'data':list(states_list)}
         response_list.extend([state_dict])
-
+    
+    assessment_list = list(Assessment.objects.annotate(value=F('name'), ).values('id','value').order_by('value'))
+    assessment_list[0]['checked'] = 'true'
```

Comment:
Ordering by name  - and then setting checked true will create some trouble if new types of assessments are added to the list.